junit5/src/main/java/cz/xtf/junit5/config/JUnitConfig.java

```diff
@@ -22,4 +25,16 @@ public static boolean cleanOpenShift() {
 			return Collections.emptyList();
 		}
 	}
+
+	public static String ciUsername() {
+		return XTFConfig.get(CI_USERNAME);
+	}
+
+	public static String ciPassword() {
+		return XTFConfig.get(CI_PASSWORD);
+	}
+
+	public static String jenkinsRerun() {
```

Comment:
boolean?

Reply:
it can be used to specify specific run URL.